doc/intro.rst

```diff
@@ -26,12 +26,18 @@ colors) with a switchable backlight:
 
 .. image:: images/st7735.jpg
 
+The HT1621 display (as purchased) supports six 7-segment characters with a 
```

Comment:
It looks like there are quite a few different variants that have 4, 5, 8 or 12 characters on the display. That statement was really just a disguised disclaimer saying it works on the 6 character device i suppose.